<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgUpdate</class>
 <widget class="QDialog" name="DlgUpdate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Version Checker</string>
  </property>
  <widget class="QLabel" name="t_latest_ver">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Latest Version: 0.0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_logo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="t_conclusion">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Conclusion: You have the latest version</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_close1">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_close2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_goto_web">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Visit AuType Webpage</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="t_note">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
